docs/conf/eu/2017/news/recap-retrospective.rst

```diff
@@ -0,0 +1,90 @@
+
+Following the conference, here is an anonymized list of CoC-related incidents that occurred during the conference:
+
+- During the conference, an attendee tweeted a joke about inappropriate alcohol use with the #writethedocs hashtag. We asked the attendee to delete the tweet, which was done immediately.
+
+- One of the presenters made a joke during their talk that was considered discriminatory towards elderly people, parents, and/or women. We removed the mention from the presentation video before publishing it, and asked that the presenter refrain from these jokes in future Write the Docs events.
+
+- We received several reports of over-use of the word "guys" to refer to people in general. This was not confined to a single incident, and repeated among presenters and attendees. Barring specific people to address, we'd like to remind everyone to carefully consider which pronouns and other classifications to use when addressing a diverse group.
+
+As mentioned, this report is anonymized and published for the sake of maintaining transparency, and to encourage folks in our community to report incidents that might disrupt the atmosphere we are passionate about.
+
+Videos
+------
+
+All talk videos are now availble on our `playlist on our YouTube channel <https://www.youtube.com/playlist?list=PLZAeFn6dfHplBYPCwJt6ItkMDt7JSgUiL>`_.
+
+We'd like to thank again all our speakers and lightning talk presenters for investing time, effort, and love into their presentations. Each year we see growth in both the diversity of topics and the spectrum of the roles that are represented in our
+conferences, and we hope to keep seeing this in years to come!
+
+Photos
+------
+
+We have photos from the event available on `Flickr <https://flic.kr/s/aHsm3Mcv9p>`_. We really love the photos each year, and you're welcome to use them for non-commercial purposes, as per our `Creative Commons
+License <https://creativecommons.org/licenses/by-nc-sa/2.0/>`_.
+
+2018 conferences
+----------------
+
+These days we are laying the groundwork for our 2018 conferences, and will announce all the information that you need to register, speak, and volunteer at the Portland and Prague events.
+
+In the meantime, save the dates for next year:
+
+- `Portland conference <http://www.writethedocs.org/conf/portland/2018/>`_, 6-8 May 2018
+- Prague conference, 9-11 September 2018
+
+We're currently looking for launch sponsors to make our events just as magical as this year's events. If you are curious about
+sponsoring any Write the Docs event, please contact us at sponsorship@writethedocs.org.
+
+Write the Docs Slack
+--------------------
+
+The Write the Docs Slack has really grown more this last year. Our general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y'all continue the conversations throughout the year. If you haven't joined, head on over and join us!
```

Comment:
ditch 'more'

Reply:
and 'general'.

Our Slack channels have been...